scripts/gen_dlaf_strong-gpu.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2023, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+# This file is an example on how to use the miniapp module.
+# Please do not add gen scripts used for benchmarks into the source repository,
+# they should be kept with the result produced.
+
+import argparse
+import miniapps as mp
+import systems
+
+system = systems.cscs["daint-gpu"]
+
+dlafpath = "<path_to_dlaf_build_dir>"
+
+run_dir = f"~/ws/runs"
+
+time = 400  # minutes
+nruns = 5
+nodes_arr = [1, 2, 4, 8, 16]
```

Comment:
please add `extra_flags = "--dlaf:bt-band-to-tridiag-hh-apply-group-size=128"` (64 for MC)

Reply:
See https://github.com/eth-cscs/DLA-Future/pull/840/commits/b573badc5bef61b3759346caf935f0f98bb192e2